src/format.rs

```diff
@@ -14,77 +14,106 @@ use std::iter::zip;
 
 /// Central function to format a file
 pub fn format_file(
-    text: &str,
+    old_tex: &str,
```

Comment:
Should this be `old_text`?

Reply:
Changing this seems to make it slower, so let's rename `old_text` to `cleaned_old_text` and `old_tex` to `old_text`?

Reply:
Thanks for catching the typo! 

After merging with `main` in [`0e4ec8c`](https://github.com/WGUNDERWOOD/tex-fmt/pull/42/commits/0e4ec8c6ce8612b08bfb7a81971a65639cd57918), I ran the benchmarks before and after correcting the typo. Both results were the same (this branch ran in ~59±2 ms for main's ~63±2 ms). Either my computer was having a good day when I ran the first benchmarks or your recent changes in main caused a little slowdown (but the new measurements for main are the same as before so I don't think it's this either). (Reverting the commit which documented the regex also didn't give 57 ms performance again.)